.dockerignore

```diff
@@ -0,0 +1,17 @@
+.github
+.vscode
+.node_modules
+.husky
+.eslintignore
+.eslintrc.cjs
```

Comment:
I'm not sure how `.dockerignore` is used exactly, but for a development container (which we'll setup later), these configuration files will be needed. They will likely be mapped to a volume that mirrors this directory.

Reply:
Do you mean you want to create a dev container where the users can open their vscode inside the container and work from there?

Reply:
Two containers (docker-compose) files, dev and prod versions.

Reply:
> Two containers (docker-compose) files, dev and prod versions.

I did that in the latest commit, but I kept the configuration files ignored in `.dockerignore` and every thing seems to run ok!